test/gurka/test_search_filter.cc

```diff
@@ -14,10 +14,10 @@ class SearchFilter : public ::testing::Test {
 
     const std::string ascii_map = R"(
```

Comment:
these test changes are needed because the previous test was falling back to astar which didnt implement the derived uturn logic. once it started usting timedep_forward which does have this logic, it was allowing a uturn at C to come back to reach 2 (the original destination). this was apparently slightly cheaper than going around the block. to keep the spirit of the test going aroudn the block, i moved the origin/destination and changed the road class to lower the speed